### DISCUSSION TITLE
Is Morse transmitted as modulated FM?

### DISCUSSION THREAD

**Discussion by n7ime** [mikaelnousiainen/RS41ng#43] [Q&A]
How is morse transmitted? 

Is this modulated FM? I am attempting to get Morse working and listening with an HT set for wide band FM. There is clearly a transmitted signal on the expected frequency, but no modulated tone like I was expecting. Without the tone, I'm having a hard time checking that it is sending the desired text.

I've attempted to set the Morse transmit string in config.c as:
```
char *cw_message_templates[] = {
//        "$cs $loc6 $altm $gs km/h $tiC",
//        "$cs $loc6",
//        "$alt m",
//        "$gs km/h $ti C",
        "TEST DE N7IME",
		NULL
};
```

I'm expecting that to just send "TEStT DE N7IME".

Can someone please clarify if I should expect to receive this as FM modulated with the Morse tone?
Can someone check me on how I have attempted to set up the text to be transmitted?

Thank you.
73,
Jason

**Comment by iz2fly** [comment]
Hello Jason. 
CW mode is not FM mode. In FM mode you receive only an unmodulated carrier.
It can be received in CW, USB, LSB mode.

**Comment by n7ime** [comment]
I wonder if it might be possible to modify this? APRS is successfully
modulating the FM signal using the SI4043_MODULATION_TYPE_FSK in
radio_si4032.c

I haven't had easy success trying to use settings from APRS for CW and PIP
in the `switch()`  starting in line 46 of that routine.

73,
Jason N7IME

**Reply by iz2fly** [reply]
With the little knowledge I have, I think that RS41 cannot modulate in a classic FM mode. In fact, FSK is not FM modulation mode.

https://www.quora.com/What-is-the-difference-between-frequency-shifting-keying-and-frequency-modulation#:~:text=FSK%20might%20be%20termed%20a,not%20necessarily)%20just%20two%20values.

**Comment by mikaelnousiainen** [comment]
@n7ime @iz2fly Si4032 can emulate FM (AFSK) using PWM to toggle the Si4032 frequency at a given rate. This is how APRS works currently. It can be used to produce an FM-modulated CW tone. You'd need to add a new mode for this -- handling should happen in a similar way to CW where the carrier is turned on/off and timing is generated by the loop.

I personally do not see this as a particularly widely used feature, so probably not something I would work on immediately... but all contributions are welcome!

**Reply by iz2fly** [reply]
More useful perhaps to add 2 variables for continuous tone and a CW pause with variable tempos, like all beacons.

**Comment by n7ime** [comment]
My use case is to reconfigure an RS41 as a hidden transmitter for
foxhunting. Our local community has recovered a number of sondes from
weather service launches,  with a few running dedicated tracking stations.
It would be fun to introduce others to recovering them, but outside of the
current sonde chasers most of us are using FM HTs for direction finding.
Modulated CW on FM would make that easier for the general ham community.

But in the meantime I can experiment using the APRS mode.  My FT1DR was not
successfully decoding the packets yet but I'll keep working at it. The
short bursts will be more difficult for manual direction finding
techniques, which is why I was preferring MCW.

73,
Jason N7IME
